java/src/test/java/com/ncc/aif/NistTA3ExamplesAndValidationTest.java

```diff
@@ -195,15 +192,13 @@ void invalidEvent() {
             @Test
             void invalidRelation() {
                 //invalid relation cluster, no importance value
-                markImportance(eventCluster, 88.0);
```

Comment:
Just a question, this would be invalid because eventCluster and relationCluster doesn't have an importance value? Not because the relationCluster doesn't have an importance value?

Reply:
This is invalid because the relationCluster doesn't have an importance. The eventCluster is created in NISTHypothesisExamples.setup(). Each setup() method is called during the constructor so you have to look at all the setup() methods to know what is fully going on. This probably needs to be made less confusing so I'm open to suggestions.